app/components/gh-members-lab-setting.js

```diff
@@ -29,16 +29,22 @@ export default Component.extend({
     feature: service(),
     config: service(),
     mediaQueries: service(),
+    ghostPaths: service(),
 
     currencies: null,
 
     // passed in actions
     setMembersSubscriptionSettings() {},
+    setStripeConnectIntegrationTokenSetting() {},
```

Comment:
This name is used throughout the components and services - is it okay?

Reply:
Could possibly remove the `Integration` part to make it a bit shorter? Otherwise seems fine 👍

Reply:
I think I'm gonna leave it a little longer for now - just to be explicit!